index.html

```diff
@@ -1,14 +1,284 @@
+        </h5>
+      </a>
+      <p class="lower-border">
+        Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptates adipisci porro consectetur exercitationem
+        corrupti facilis quod ipsa deleniti. Dolore, ea. At, necessitatibus saepe odio cupiditate enim ducimus ipsam
+        illo.
+        Dignissimos?
+      </p>
+
+      <!-- Blog Post Links -->
+      <a href="/blog-post.html"><button>
+          JULY 24
+        </button></a>
+      <a href="/blog-post.html">
+        <h5>
+          SUPPORT GROUPS FOR FAILURE
+        </h5>
+      </a>
+      <p class="paragraph-padding">
+        Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptates adipisci porro consectetur exercitationem
+        corrupti facilis quod ipsa deleniti. Dolore, ea. At, necessitatibus saepe odio cupiditate enim ducimus ipsam
+        illo.
+        Dignissimos?
+      </p>
+
+      <!-- Learn-More Article -->
+      <a href="/learn-more.html">
+        <h5 class="double-border-section">
+          GLOAT ONLY AFTER VICTORY
+        </h5>
+      </a>
+      <div class="pic-float">
+        <img src='/images/hexus.jpg' class="pic-shrink">
+      </div>
+      <p>
+        Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptates adipisci porro consectetur exercitationem
+        corrupti facilis quod ipsa deleniti. Dolore, ea. At, necessitatibus saepe odio cupiditate enim ducimus ipsam
+        illo.
+        Dignissimos?
+      </p>
+      <a href="/learn-more.html"><button class="medium-button">
+          LEARN MORE
+        </button></a>
+
+      <!-- Learn-More Article -->
+      <a href="/learn-more.html">
+        <h5 class="higher-border">
+          UNDERATED WEOPONRY
+        </h5>
+      </a>
+      <div class="pic-float">
+        <img src='/images/Harley.webp' class="pic-shrink">
+      </div>
+      <p>
+        Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptates adipisci porro consectetur exercitationem
+        corrupti facilis quod ipsa deleniti. Dolore, ea. At, necessitatibus saepe odio cupiditate enim ducimus ipsam
+        illo.
+        Dignissimos?
+      </p>
+      <button class="medium-button">
+        LEARN MORE
+      </button>
+    </section>
+  </section>
+
+  <!-- Contact Info Footer -->
+  <section id="footer">
+    <h5 class="centered-titles">
+      QUESTIONS OR COMMENTS? GET IN TOUCH:
+    </h5>
+
+    <!-- Footer Flex-Box -->
+    <section id="footer-box">
+
+      <!-- Footer Left Column -->
+      <section id="footer-left">
+        <input type="text" placeholder="Name" class="input" id="name-input">
+        <input type="text" placeholder="Email" class="input" id="email-input">
+        </div>
+        <section id="footer-bottom-left">
+          <textarea type="text" placeholder="Message" id="message-box"></textarea>
+          <button id="message-button">
+            SEND MESSAGE
+          </button>
+        </section>
+      </section>

```

Comment:
This section is a great start, The layout provided has the `<textarea>` stretching all the way down so that the bottom of your button lines up with the bottom of the contact information on the right side. You can add the attributes `col="(number)"` and `row="(number)"` to adjust the sizes of a text area. 

This comment also pertains to your other pages as well